src/web/app/components/response-time-series-chart/response-time-series-chart.component.html

```diff
@@ -8,8 +8,8 @@
             <option *ngFor="let duration of [1, 5, 10, 15, 30, 60]" [ngValue]="duration">{{ duration + ' min'}}</option>
         </select>
         <label>Interval:</label>
-        <select class="form-control" [ngModel]="model.intervalMinutes" (ngModelChange)="setIntervalHandler($event)">
-            <option *ngFor="let interval of [1, 5, 10, 15, 30, 60]" [ngValue]="interval">{{ interval + ' min'}}</option>
+        <select class="form-control" [ngModel]="model.intervalSeconds" (ngModelChange)="setIntervalHandler($event)">
```

Comment:
I see, so this interval is actually the refresh interval? I think can make a fixed view too.